src/services/ai-service/ai-service.types.ts

```diff
@@ -38,10 +38,13 @@ export interface WriteResponse {
 }
 
 export interface AIServiceError {
-  code: number;
+  type: AIServiceErrorTypes;
   message: string;
+  code: number;
 }
 
 export enum AIServiceErrorTypes {
   AI_SERVICE_ERROR = 'AI_SERVICE_ERROR',
+  FILTERED_RESPONSE = 'FILTERED_RESPONSE', // this was just added for testing purposes
```

Comment:
Please add a prefix to all error names to mark them as coming from the AI Service. We might reuse this approach for other services.

Reply:
the backend team provided the ones to use